aten/src/ATen/native/mkl/SparseBlasImpl.cpp

```diff
@@ -367,6 +367,15 @@ void addmm_out_sparse_csr(
         alpha,
         result.transpose(0, 1));
   }
+  if (mat1.layout() == kStrided &&
+      (mat2.layout() == kSparseCsc || mat2.layout() == kSparseBsc)) {
+    return addmm_dense_result(
+        mat2.transpose(0, 1),
+        mat1.transpose(0, 1),
+        beta,
+        alpha,
+        result.transpose(0, 1));
```

Comment:
Nit:
```suggestion
        mat2.transpose(-2, -1),
        mat1.transpose(-2, -1),
        beta,
        alpha,
        result.transpose(-2, -1));
```

Reply:
The code of the method looks like it might benefit from some refactoring...